hal/imx_rt.c

```diff
@@ -244,44 +274,134 @@ const flexspi_nor_config_t __attribute__((section(".flash_config"))) qspiflash_c
 };
 #endif
 
-
+/** Flash configuration in the .flash_config section of flash **/
 #ifdef CPU_MIMXRT1052DVJ6B
-    #define CONFIG_FLASH_SIZE (64 * 1024 * 1024)  /* 64MBytes  */
-    #define CONFIG_FLASH_PAGE_SIZE 512UL          /* 512Bytes  */
-    #define CONFIG_FLASH_SECTOR_SIZE (256 * 1024) /* 256KBytes */
-    #define CONFIG_FLASH_BLOCK_SIZE (256 * 1024)  /* 256KBytes */
-    #define CONFIG_FLASH_UNIFORM_BLOCKSIZE true
-    #define CONFIG_SERIAL_CLK_FREQ kFlexSpiSerialClk_30MHz
-const flexspi_nor_config_t __attribute__((section(".flash_config"))) qspiflash_config = {
```

Comment:
@bigbrett do you think I should keep a version available for HyperFlash also? I think it would be fairly easy to support it still and test.

Reply:
Yes it is probably a smart idea, since it is the default flash device on the EVKs, and it could see increased adoption over time. Looks like the demo apps  contain the LUT and flash config to make it work too.